src/components/CityWeatherShortcut/CityWeatherShortcut.tsx

```diff
@@ -17,9 +18,10 @@ import { CityWeatherShortcutLoader } from './components/CityWeatherShortcutLoade
 
 type Props = {
   cityName: string;
+  type: ButtonTypes;
 };
 
-const CityWeatherShortcut = ({ cityName }: Props) => {
+const CityWeatherShortcut = ({ cityName, type }: Props) => {
```

Comment:
**question** What do you think about setting default `type` to heart?

Reply:
Okey. Makes no difference to me

Reply:
But on the other hand it probably won't pass manual tests because on mockups there is cross and we will have heart

Reply:
If you pass cross as type, then it won't be overwritten by the default value and tests should pass. Anyway, it's just a suggestion and you don't have to implement that :)

Reply:
Okay, I can add it, if you think it is more correct. As I said it doesn't make any difference to me, but maybe you are right, @bednarskimichal ?